ziti-agent/cmd/webhook/pods.go

```diff
@@ -430,19 +432,19 @@ func (zh *zitiHandler) handleDelete(ctx context.Context, pod *corev1.Pod, respon
 
 	if zh.Config.ZitiType == zitiTypeRouter {
 
-		if err := zh.ZC.deleteZitiRouter(context.Background(), pod.Name); err != nil {
+		if err := zh.ZC.deleteZitiRouter(ctx, pod.Name); err != nil {
 			return failureResponse(response, err)
 		}
 
 	} else {
 
-		name, ok := hasContainer(pod.Spec.Containers, fmt.Sprintf("%s-%s", pod.Labels["app"], zh.Config.Prefix))
```

Comment:
app label doesn't match how the name is built now, so we can either match on prefix alone or rebuild the name by calling buildZitiIdentityName()